src/selectPackage/selectPackage.ts

```diff
@@ -1,3 +1,26 @@
+/* eslint-disable linebreak-style */
 import "./selectPackage.scss";
+
 export class SelectPackage {
+  // Declare state variables
+  benefits: Array<string>;
+  discounts: { [plan:string]:{discount: number, benefits: Array<string>}};
```

Comment:
Why this complication in the ts?  Can this just be hardcoded in the HTML?  Same with benefits.

Reply:
aside from the complication of add'l code, realize that aurelia will create observables on these unless you specify them to be "one-time".  but then again, it begs the question why have them at all?